go/gorilla/security/audit/handler-assignment-from-multiple-sources.yaml

```diff
@@ -0,0 +1,52 @@
+rules:
+- id: handler-assignment-from-multiple-sources
+  options:
+    taint_unified_mvars: true
+  metadata:
+    cwe:
+    - 'CWE-289: Authentication Bypass by Alternate Name'
+    category: security
+    technology:
+    - gorilla
+    confidence: MEDIUM
+    license: Commons Clause License Condition v1.0[LGPL-2.1-only]
+    references:
+    - https://cwe.mitre.org/data/definitions/289.html
+    subcategory:
+    - vuln
+    impact: MEDIUM
```

Comment:
I wrote this rule forever ago based on a haphazard conversation. I'm changing impact and likelihood to "LOW" to match my belief about the rule